common/static/coffee/spec/discussion/view/discussion_thread_view_spec.coffee

```diff
@@ -102,40 +157,59 @@ describe "DiscussionThreadView", ->
                 course_settings: DiscussionSpecHelper.makeCourseSettings()
             )
 
+        describe "responses", ->
+            it "can post a first response", ->
+                renderWithTestResponses(@view, 0)
+                postResponse(@view, 1)
+                expect(@view.$(".forum-response").length).toBe(1)
+                # can't edit the main post (by default test setting)
+                expect(@view.$(".action-edit").parent(".is-hidden").length).toBe(1)
+
+            it "can post a second response", ->
+                renderWithTestResponses(@view, 1)
+                expect(@view.$(".forum-response").length).toBe(1)
+                # can't edit the main post (by default test setting)
+                expect(@view.$(".action-edit").parent(".is-hidden").length).toBe(1)
+
+                postResponse(@view, 2)
```

Comment:
Same comment above-- be more specific about testing the menus so the state of each item (initial post, responses) is verified.